src/game/behaviors/snowman.inc.c

```diff
@@ -155,7 +155,7 @@ void bhv_snowmans_bottom_loop(void) {
 
 void bhv_snowmans_head_init(void) {
     u8 starFlags = save_file_get_star_flags(gCurrSaveFileNum - 1, COURSE_NUM_TO_INDEX(gCurrCourseNum));
-    s8 behParams = GET_BPARAM1(o->oBehParams);
+    u8 behParams = GET_BPARAM1(o->oBehParams);
```

Comment:
i think there are other objects that need this too, unagi comes to mind, did you make sure to check everything that uses bparam1?

Reply:
I did a search on everything `BPARAM1` and also a regex search against `oBehParams` and `>> 24`.

Where does unagi even use bparam1? I don't see anything for it in the behavior file.

Reply:
it should be fine then, basically any object that spawns a star uses its bparam1 for the star id but i think this is handled in the star spawn function based off gcurrentobject